### ISSUE TITLE
I'd like to create merge/pull requests to another branch

### ISSUE THREAD

**Issue by PetrSnobelt** [renovatebot/config-help#41]
Hi, 
I'd like to create merge requests to the different branch, not to  "master" because I'd like to let QA test this branch before merging into "master".
Is it possible, or there is another workflow for that?

**Comment by rarkins** [comment]
Yes, try the `baseBranch` option. e.g. `"baseBranch": "next"`. This will mean all PRs will target `next` and not `master`. Make sure it exists - Renovate won't create it.

**Comment by PetrSnobelt** [comment]
Super, I will try. 
Why this option is not mentioned in `renovate --help`
